app/models/product.rb

```diff
@@ -214,13 +214,16 @@ def stage
       raise
     end
 
+    # Writes out a manifest to the router path unless the repository has been newly cloned.
+    #
+    # @return [true] always returns true
     def write_manifest
-      File.open(router.path.join('manifest.yml'), 'w') do |f|
-        f.write({
-          'name' => title,
-          'description' => description
-        }.to_yaml)
-      end if (!title.blank? || !description.blank?) || !router.path.join('manifest.yml').exist?
+      manifest = Manifest.load( router.path.join('manifest.yml') )
+
+      manifest = manifest.merge({ name: title, description: description })
+
+      manifest.save(router.path.join('manifest.yml')) if (!title.blank? || !description.blank?) || !router.path.join('manifest.yml').exist?
```

Comment:
`router.path.join('manifest.yml')` is duplicated here: https://github.com/OSC/ood-dashboard/blob/ea579eccff193a8b45f6e6d98b1aa5bcfdf385f2/app/apps/ood_app.rb#L139-L143

```ruby
  def load_manifest
    default = path.join("manifest.yml")
    alt = path.dirname.join("#{path.basename}.yml")
    alt.exist? ? Manifest.load(alt) : Manifest.load(default)
  end
```

Should add a public method OodApp#manifest_path and then update OodApp#manifest to look like this:

```diff

  def manifest
-    @manifest ||= load_manifest
+    @manifest ||= Manifest.load(manifest_path)
  end
```

The whole alt manifest thing in `OodApp#load_manifest` can go away. That is an artifact from the old dashboard.